packages/kit/src/views/Market/MarketDetailV2/components/InformationTabs/components/TransactionsHistory/TransactionItem.tsx

```
@@ -0,0 +1,144 @@
+    copyText(item.owner);
+  };
+
+  // Display relative time like "3 minutes" / "2 hours" without suffix.
+  const formatRelativeTime = (timestamp: number) =>
+    formatDistanceToNowStrict(timestamp * 1000, {
+      addSuffix: false,
+      roundingMethod: 'floor',
+    });
+
+  const formatAmount = (amount: string) => {
+    const num = parseFloat(amount);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)